README.md

```diff
@@ -22,7 +22,7 @@ protractor-flake
 
 # Full options
 # protractor-flake <protractor-flake-options> -- <options to be passed to protractor>
-protractor-flake --protractor-path=/path/to/protractor --parser standard --node-bin node --max-attempts=3 -- path/to/protractor.conf.js
+protractor-flake --protractor-path=/path/to/protractor --parser standard --node-bin node --max-attempts=3 --color=magenta -- path/to/protractor.conf.js
```

Comment:
The color option works in the following way:
`--color=true` makes flake messages of `DEFAULT_COLOR` (which is magenta)
`--color=green` makes flake messages green

Letting the users decide what color they want does not seem like a bad idea to me - we need to respect their preferences, don't we :-)